src/MessagePack.UnityClient/Assets/Scripts/MessagePack/SequencePool.cs

```diff
@@ -12,6 +13,15 @@ namespace MessagePack
     /// </summary>
     internal class SequencePool
     {
+        /// <summary>
+        /// The value to use for <see cref="Sequence{T}.MinimumSpanLength"/>.
+        /// </summary>
+        /// <remarks>
+        /// Individual users that want a different value for this can modify the setting on the rented <see cref="Sequence{T}"/>
+        /// or by supplying their own <see cref="IBufferWriter{T}" />.
+        /// </remarks>
+        private const int MinimumSpanLength = 64 * 1024;
```

Comment:
@neuecc Can we perhaps leave this at 32KB or smaller and crank it up to 64KB from the LZ4 functions?
From your research in #680 it looks like 32KB might actually be the sweet spot.

Reply:
Compression depends on data, so we don't say always 32K is the sweet spot.
However, there is no doubt that larger is always better than smaller.
Also, you can afford twice as much.
Of course, the memory will also double, but ...

Reply:
I'm just saying that if compression is the primary reason we would go so large as 64KB, can't we choose something smaller here and then in the compression methods raise the number just for those portions?

Reply:
I think we can.
However, if it was shared on Pool, I thought it would be better not to increase the number of buckets.

Reply:
> I thought it would be better not to increase the number of buckets.

That's a good point. I guess let's leave it as-is for now (at 64KB) per your recommendation and we can adjust later if necessary without it being a breaking change.